### DISCUSSION TITLE
Fresh install fails with Table 'analytics' already exists

### DISCUSSION THREAD

**Discussion by gurumark** [requarks/wiki#3362] [Help / Questions]
**Describe the bug**
Fresh install from docker fails to start

**To Reproduce**
docker run -d -p 8080:3000 --name wiki --restart unless-stopped -e "DB_TYPE=mysql" -e "DB_HOST=db" -e "DB_PORT=3306" -e "DB_USER=wikijs" -e "DB_PASS=wikijsrocks" -e "DB_NAME=wiki" requarks/wiki
**Expected behavior**
The container should start.

**Screenshots**
Loading configuration from /wiki/config.yml... OK
2020-03-07T13:15:21.336Z [MASTER] info: =======================================
2020-03-07T13:15:21.338Z [MASTER] info: = Wiki.js 2.1.113 =====================
2020-03-07T13:15:21.338Z [MASTER] info: =======================================
2020-03-07T13:15:21.338Z [MASTER] info: Initializing...
2020-03-07T13:15:21.602Z [MASTER] info: Using database driver mysql2 for mysql [ OK ]
2020-03-07T13:15:21.606Z [MASTER] info: Connecting to database...
2020-03-07T13:15:21.632Z [MASTER] info: Database Connection Successful [ OK ]
migration file "2.0.0.js" failed
migration failed with error: create table `analytics` (`key` varchar(255) not null, `isEnabled` boolean not null default '0', `config` json not null) default character set utf8mb4 - Table 'analytics' already exists
2020-03-07T13:15:21.683Z [MASTER] error: Database Initialization Error: create table `analytics` (`key` varchar(255) not null, `isEnabled` boolean not null default '0', `config` json not null) default character set utf8mb4 - Table 'analytics' already exists

**Host Info (please complete the following information):**
 - OS: [ Docker]
 - Wiki.js version: [2.1.113]
 - Database engine: [ mysql 5.7]

**Comment by NGPixel** [comment]
The database you're trying to install to is not empty. Drop the database, create a new one and try again.

**Comment by gurumark** [comment]
I dropped all the tables and tried again. The same thing. Sorry, I should have added this in the description.

**Comment by Smankusors** [comment]
`DB_HOST=db`, is this... the database from the hostname `db`? Or is this a mysql docker container?

**Comment by NGPixel** [comment]
@Smankusors `db` would refer to another docker container with name `db` in this example.

@gurumark Make sure you don't run the instance twice? Somehow the migration script runs twice. So either you have 2 instances running at the same time, or the migrations table mysteriously get deleted.

**Comment by gurumark** [comment]
@NGPixel This is a fresh Install on an empty database. I think there is a bug in the script creating db tables. Everytime I run the container ( after dropping the tables it created) it gets stuck in the same table. I think it's trying to create the same table it already created and errors out because it exists there then the execution stops because of the error.

**Comment by Smankusors** [comment]
@NGPixel I think it's weird because the command OP used, there's no network linking. Is that possible? Without `docker network connect`? Or OP set it on `/etc/hosts`?

@gurumark you still didn't tell us about your database. Is your database is a mysql docker container or the database from another computer? Who is `db` on `DB_HOST` you refer is?

**Comment by gurumark** [comment]
DB-HOST is a mysql server running on the host computer.

**Comment by Smankusors** [comment]
I can't replicate this. Can you try with docker-compose?

**Comment by Smankusors** [comment]
can you try with Wiki.js 2.2.51 see if this still happens?

**Comment by REOLDEV** [comment]
Hi, same problem under Plesk Docker.
`migration file "2.0.0.js" failed
migration failed with error: create table `analytics` (`key` varchar(255) not null, `isEnabled` boolean not null default '0', `config` json not null) default character set utf8mb4 - You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'json not null) default character set utf8mb4' at line 1
2020-04-10T14:20:51.800Z [MASTER] [31merror[39m: Database Initialization Error: create table `analytics` (`key` varchar(255) not null, `isEnabled` boolean not null default '0', `config` json not null) default character set utf8mb4 - You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'json not null) default character set utf8mb4' at line 1
`

Already cleared the Database
DB Host is on the Hostserver

**Comment by eeshugerman** [comment]
@REOLDEV that is a different error than OP's. Yours is a syntax error. What database are you using?

**Comment by Smankusors** [comment]
> @REOLDEV that is a different error than OP's. Yours is a syntax error. What database are you using?

it's solved on #1696
